internal/providers/terraform/ibm/en_subscription.go

```diff
@@ -0,0 +1,42 @@
+package ibm
+
+import (
+	"github.com/infracost/infracost/internal/resources/ibm"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+func getEnSubscriptionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:                "ibm_en_subscription_android",
```

Comment:
Yeah, the resource is `ibm_en_subscription_android`, however Go _really_ did not like that files had the word Android in them. It kept saying that it could not find the functions that were in the files named that, so that's why for this one it doesn't quite match